CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape.uc

```diff
@@ -902,7 +920,17 @@ simulated function OnReceiveFocus()
 		{
 			`XSTRATEGYSOUNDMGR.PlayGeoscapeMusic(); // Otherwise SS music doesn't stop after confirmation
 			SSManager = none;
+
+			if (GetAction().bNewAction)
+			{			
+				NewGameState = class'XComGameStateContext_ChangeContainer'.static.CreateChangeState("Turn off covert action NEW flag");
+				ActionState = XComGameState_CovertAction(NewGameState.ModifyStateObject(class'XComGameState_CovertAction', ActionRef.ObjectID));
+				ActionState.bNewAction = false;
+				`XCOMGAME.GameRuleset.SubmitGameState(NewGameState);
+			}
+
 			UpdateList();
+			AttemptSelectAction(ActionRef);
```

Comment:
I'm pretty sure this won't work since `ActionRef` will be reset to the first one in `UpdateList()` - you need to save ref to old action before you call `UpdateList()`